guides/v2.1/javascript-dev-guide/widgets/widget_accordion.md

```diff
@@ -179,6 +179,44 @@ $( "#element" ).accordion( "deactivate", 1 );
 $( "#element" ).accordion( "deactivate", [0,1]);
 ```
 
+## Events {#accordion_events}
+
+The accordion widget subscribes to the same events as the [Tabs widget]({{page.baseurl}}/javascript-dev-guide/widgets/widget_tabs.html):
+
+-   [beforeOpen callback](#accordion_beforeOpen)
```

Comment:
@serhiyzhovnir,  I forgot to ask you to update these anchor links when I asked you to remove them from the headings below. That's what's causing the build to fail. You'll need to update these anchors.

- `[beforeOpen callback](#beforeopen-callback)`
- `[dimensionsChanged](#dimensionschanged)`